lib/openlogi/base_object.rb

```diff
@@ -14,7 +14,9 @@ def initialize(attributes = {}, &block)
       super
     end
 
+    property :error
     property :errors
+    property :error_description
```

Comment:
Testing this now I realized that we need these properties on BaseObject otherwise we will issue warnings when assigning attributes to objects from the response.